modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -112,11 +112,19 @@ event(#postback{message={delete_all, Args}}, Context) ->
                     z_render:growl(?__("Sorry, you are not allowed to delete this.", Context), Context)
```

Comment:
Line 112 is too long:                     z_render:growl(?__("Sorry, you are not allowed to delete this.", Context), Context).